tests/unit/test_e2e.py

```diff
@@ -1013,7 +1042,13 @@ def test_provider_get_maintenances(
             else:
                 data.add_data_part(data_type, file_obj.read())
 
-    parsed_notifications = provider_class().get_maintenances(data)
+    try:
+        parsed_notifications = provider_class().get_maintenances(data)
+    except ProviderError as exc:
```

Comment:
should not install all the necessary packages for testing?

Reply:
I can do that locally, but Github CI is not installing all packages.  This is how I worked around that.

Reply:
we should update this https://github.com/networktocode/circuit-maintenance-parser/blob/develop/.github/workflows/ci.yml#L132 and include the necessary dependencies for the tests